src/Tribe/Ignored_Events.php

```diff
@@ -975,7 +977,35 @@ public function hook() {
 			 */
 			tribe_notice( 'legacy-ignored-events', array( $this, 'render_notice_legacy' ), 'dismiss=1&type=warning' );
 
+			add_filter( 'wp_count_posts', [ $this, 'patch_count_posts' ], 10, 3 );
+
 			return true;
 		}
+
+		/**
+		 * Patch post-list-table queries to include ignored status,
+		 * as WP just accesses the stati without checking if they exist.
+		 *
+		 * @see wp-admin/includes/class-wp-posts-list-table.php->get_views()
+		 *
+		 * @since TBD
+		 *
+		  * @param object $counts       An object containing the current post_type's post
```

Comment:
* :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Expected 9 space(s) before asterisk; 10 found

<sub>hash: 5b73f0ece26cc3422599e45ae84b2c7a</sub>